geoutils/georaster/raster.py

```diff
@@ -1375,13 +1385,13 @@ def crop(
     def reproject(
         self: RasterType,
         dst_ref: RasterType | rio.io.Dataset | str | None = None,
-        dst_crs: CRS | str | None = None,
+        dst_crs: CRS | str | int | None = None,
```

Comment:
why is int needed? For EPSG codes? If so, it should be clarified in the docstring what argument can be passed.

Reply:
Done